src/Apps/Sell/SellFlowContext.tsx

```diff
@@ -185,7 +185,8 @@ export const SellFlowContextProvider: React.FC<SellFlowContextProviderProps> = (
   }
 
   const state = {
-    isFirstStep: index === 0,
+    // Both the first and second steps are considered the first step because we don't want to allow going back to the artist step.
+    isFirstStep: index === 0 || index === 1,
```

Comment:
We don't want to display the "Back" button on the Title step to avoid having a button to go back to change the artist.

I'm not so sure about the new definition of `isFirstStep`, but I thought it makes sense because depending on whether you're creating or editing a submission, the first step is either the "Artist" step or the "Title" step 🤔

Reply:
Just curious - why are we not allowing this? I can imaging user clicking on a wrong artist by mistake and going back to revert it

Reply:
Let's phrase it like this: we still allow it (by using the browser's back button) but decided not to provide the "Back" button in the form on the step after the "Artist" step.

Reply:
Like this, we allow changing the artist and the back button keeps its normal behavior. But we don't provide a clear path to updating the artist of an already existing submission.